plugins/shrs_mux/src/python/error_theme.rs

```diff
@@ -0,0 +1,51 @@
+use crossterm::style::ContentStyle;
+use lazy_static::lazy_static;
+use regex::Regex;
+use shrs::{crossterm::Stylize, prelude::styled_buf::StyledBuf, readline::SyntaxTheme};
+lazy_static! {
+    static ref FILE_PATTERN: Regex = Regex::new(r#"File (".*"), line (\d+), in (.*)"#).unwrap();
+    static ref ERROR_PATTERN: Regex = Regex::new(r#"([^:]+:) (.+)"#).unwrap();
+}
+
+pub struct PythonErrorTheme {
+    pub error_style: ContentStyle,
+    pub line_style: ContentStyle,
+    pub file_style: ContentStyle,
+    pub reason_style: ContentStyle,
+}
+
+impl PythonErrorTheme {
+    pub fn new() -> Self {
+        Self {
+            error_style: ContentStyle::new().red(),
+            line_style: ContentStyle::new().dark_blue(),
+            file_style: ContentStyle::new().green(),
+            reason_style: ContentStyle::new().yellow(),
+        }
+    }
+}
+impl SyntaxTheme for PythonErrorTheme {
+    fn apply(&self, buf: &mut StyledBuf) {
+        let content = buf.content.clone();
+        if content.starts_with("Traceback") {
```

Comment:
hm, since python should write it's errors to stderr, i think we can just highlight everything from stderr using this error theme instead of relying on a special string - can implement in a follow up issue though